### PR TITLE
check palindrome in a number in Java

### LINKED ISSUE #1718: Pallindromes in Java/Python
Hi! I want to contribute by writing code for this program. It will:
1. Check if a number is palindrome or not
2.Check if a name/word is palindrome or not
Plz assign me this as a participant of GSSOC20.
@SakshiShreya @singh-shreya6 @asha15 @raghav-dalmia

### PR DESCRIPTION
**Pull request by raksha009**
Fixes #1718 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added a new code
- 

#### Languages Used:
- Java
- 

#### Files Added:
- palindrome_no.java
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Palindrome/palindrome_no.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// Java program to check if a number entered is a palindrome
+
+import java.util.*;
+
+public class palindrome_no {
+	
+	// Find reverse of a number
+	public static int reverse(int number) {
```
Comment (raghav-dalmia) [inline Palindrome/palindrome_no.java]: use 4 space indentation instead of using tabs

Comment (raksha009) [reply Palindrome/palindrome_no.java]: @raghav-dalmia  I have made the changes. Plz review

**Comment by raksha009** [comment]
@asha15 @singh-shreya6 @somya-kapoor plz review my PR

File: Palindrome/palindrome_no.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import java.util.*;
+
+public class palindrome_no {
+	
+    // Find reverse of a number
+    public static int reverse(int number) {
+        int reverseno = 0 ;
+		
+	while (number >0) {
+	    reverseno = (int) (reverseno*Math.pow(10, 1) + number%Math.pow(10, 1));
```
Comment (raghav-dalmia) [inline Palindrome/palindrome_no.java]: add space around operators (>*</=+).

**Comment by raghav-dalmia** [comment]
Also, squash the commits.

**Comment by raksha009** [comment]
done with squashing
@raghav-dalmia

**Comment by raksha009** [comment]
@raghav-dalmia i have squashed all these new commits also. Its not showing below the new commits. But in my terminal, i have done them and it shows "Everything up-to-date"

**Comment by raghav-dalmia** [comment]
Refer [this ](https://github.com/wprig/wprig/wiki/How-to-squash-commits) and [this](https://stackoverflow.com/questions/999907/git-push-says-everything-up-to-date-even-though-i-have-local-changes).

**Comment by raksha009** [comment]
done @raghav-dalmia plz merge. thanks

**Comment by raksha009** [comment]
@raghav-dalmia plz review my Pr

**Pull request merged**